### PR TITLE
Ci/add full ci workflow

### PR DESCRIPTION
**Pull request by neodmy**
## Description

Implement pipeline to release and publish packages. See `RELEASE_INSTRUCTIONS.md` for a detailed explanation.

## Related Issue

#77 

## Motivation and Context

See `RELEASE_INSTRUCTIONS.md` for motivations.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: .github/workflows/version.yml [hunk 1]
```diff
@@ -36,7 +46,27 @@ jobs:
...
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
+        with:
+          branch: releases/bump-version-v${{ steps.extract-lerna-version.outputs.lerna-version }}
+          title: releases/bump-version-v${{ steps.extract-lerna-version.outputs.lerna-version }}
+          author: github-actions[bot] <github-actions[bot]@users.noreply.github.com>
+          commit-message: "chore: bump version to v${{ steps.extract-lerna-version.outputs.lerna-version }}"
+          labels: releases
+          body: |
+            Bump packages to v${{ steps.extract-lerna-version.outputs.lerna-version }}. A new draft release will be generate on merge.
```
Comment (inigomarquinez) [inline .github/workflows/version.yml:67]: Just one typo:

```suggestion
            Bump packages to v${{ steps.extract-lerna-version.outputs.lerna-version }}. A new draft release will be generated on merge.
```

File: RELEASE_INSTRUCTIONS.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Release Instructions
+
+## TL;DR
+
+1. Trigger the `Version` workflow specifying the `version-bump`
```
Comment (inigomarquinez) [inline RELEASE_INSTRUCTIONS.md:5]: Perhaps we can add the link to the workflow: https://github.com/onebeyond/cuckoojs/actions/workflows/version.yml

**Review by inigomarquinez — CHANGES_REQUESTED** [review]
Awesome, @neodmy !

I've added a couple of suggestions 😬

**Review by UlisesGascon — APPROVED** [review]

**Review by inigomarquinez — APPROVED** [review]

**Pull request merged**